### PR TITLE
error fixed #166

### LINKED ISSUE #166: Bug: Reload button not reloading the page
**Describe the bug**

Reload button not reloading if an error occurs while fetching resources

**To Reproduce**

It should reload the page and try to fetch resources again
**Screenshots**

How the screen looks like when such an error happens
image

**Desktop (please complete the following information):**
 

**Smartphone (please complete the following information):**
 

**Additional context**

### PR DESCRIPTION
**Pull request by Parth1125**
error fixed now reload button is working

### CODE REVIEW

**Comment by Ngoakor12** [comment]
Thanks for the contribution @Parth1125 :muscle: 

Next time please follow the [contributing guidelines](https://github.com/Ngoakor12/coding-resource-finder/blob/master/CONTRIBUTING.md):
- Ask to work on an issue first
- And just make sure that the PR title is descriptive
- Use the PR template provided to you

File: frontend/src/pages/ErrorFetchingResources/ErrorFetchingResources.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ export default function ErrorFetchingResources() {
   const navigate = useNavigate();
 
   function navigateToHomepage() {
-    navigate("/");
+    // navigate("/");
```
Comment (Ngoakor12) [inline frontend/src/pages/ErrorFetchingResources/ErrorFetchingResources.js]: Please rather remove the unused code. Same thing for unused variables and imports

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
one tiny thing...

File: frontend/src/pages/ErrorFetchingResources/ErrorFetchingResources.js (hunk 1 above)
Comment (Parth1125) [reply frontend/src/pages/ErrorFetchingResources/ErrorFetchingResources.js]: Ok sure

**Comment by Parth1125** [comment]
> Thanks for the contribution @Parth1125 💪
> 
> Next time please follow the [contributing guidelines](https://github.com/Ngoakor12/coding-resource-finder/blob/master/CONTRIBUTING.md):
> 
> * Ask to work on an issue first
> * And just make sure that the PR title is descriptive
> * Use the PR template provided to you

sure👍

**Review by Ngoakor12 — APPROVED** [review]

**Comment by Ngoakor12** [comment]
Looks good, thanks again!

**Pull request merged**